package.json

```diff
@@ -104,7 +104,7 @@
     "xss-clean": "0.1.1"
   },
   "devDependencies": {
-    "acorn": "6.1.1",
+    "acorn": "^7.1.1",
```

Comment:
Dependencies need to be fixed version (Remove ^)

Reply:
updated